doc/templates/index.html

```diff
@@ -0,0 +1,256 @@
+{% extends "layout.html" %}
+{% set title = 'scikit-learn: machine learning in Python' %}
+{% block content %}
+<div class="container-fluid sk-landing-bg py-3">
+  <div class="container sk-landing-container">
+    <div class="row">
+      <div class="col-md-6 mb-3 mb-md-0">
+        <h1 class="sk-landing-header text-white text-monospace">scikit-learn</h1>
+        <h4 class="sk-landing-subheader text-white font-italic mb-3">Machine Learning in Python</h4>
+        <a class="btn sk-landing-btn mb-1" href="{{ pathto('tutorial/basic/tutorial') }}" role="button">Quick Start</a>
+        <a class="btn sk-landing-btn mb-1" href="whats_new.html" role="button">Whats New in {{ version }}</a>
+        <a class="btn sk-landing-btn mb-1" href="{{ pathto('documentation') }}" role="button">Documentation</a>
+      </div>
+      <div class="col-md-6 d-flex">
+        <ul class="sk-landing-header-body">
+          <li>Simple and efficient tools for predictive data analysis</li>
+          <li>Accessible to everybody, and reusable in various contexts</li>
+          <li>Built on NumPy, SciPy, and matplotlib</li>
+          <li>Open source, commercially usable - BSD license</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container sk-landing-container pt-3 body" role="main">
+  <div class="row no-gutters">
+    <div class="col-md-4 mb-3 px-md-2 sk-px-xl-4">
+      <div class="card h-100">
+        <div class="card-body">
+          <a href="supervised_learning.html#supervised-learning"><h4 class="sk-card-title card-title">Classification</h4></a>
+          <p class="card-text">Identifying to which category an object belongs to.</p>
+          <p class="card-text"><strong>Applications:</strong> Spam detection, image recognition.</br>
+          <strong>Algorithms:</strong>
+          <a href="modules/svm.html#svm-classification">SVM</a>,
+          <a href="modules/neighbors.html#classification">nearest neighbors</a>,
+          <a href="modules/ensemble.html#forest">random forest</a>,
+          and <a href="supervised_learning.html#supervised-learning">more...</a></p>
+        </div>
+        <div class="overflow-hidden mx-2 text-center flex-fill">
+          <a href="auto_examples/classification/plot_classifier_comparison.html"  aria-label="Classification">
+          <img src="_images/sphx_glr_plot_classifier_comparison_001_carousel.png" class="sk-index-img" style="width: unset;" alt="Classification">
```

Comment:
Actually for the alt text it would be better to be more informative about what the image is actually about (e.g. the title of the specific example rather than the title of the section).